.github/ISSUE_TEMPLATE/bug_report_form.yml

```
@@ -0,0 +1,167 @@
+name: "Calendar Bug"
+description: "Have you encountered a bug?"
+labels: ["bug", "0. to triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### 👍Thanks for reporting issues back to Nextcloud! 
+        This is the issue tracker of Nextcloud, if you have any support question please check out https://nextcloud.com/support
+        
+        This is the bug tracker for the web-calendar component. Find other components at https://github.com/nextcloud/
+        Everything that is related to the CalDAV server, meaning it does not only affect the web-calendar but also your mobile and desktop clients, belongs in the Server repository: https://github.com/nextcloud/server
+        
+        For reporting potential security issues please see https://nextcloud.com/security/
+        
+        To make it possible for us to help you please fill out below information carefully. 
+        You can also use the Issue Template application to prefill most of the required information: https://apps.nextcloud.com/apps/issuetemplate
```

```suggestion
        You can also use [the Issue Template application](https://apps.nextcloud.com/apps/issuetemplate) to prefill most of the required information.
```

Is this still recommended and up-to-date?

---

[Last supported version is NC 20](https://apps.nextcloud.com/apps/issuetemplate), so I'm guessing the sentence can be removed.

---

you said it :D